src/components/block/DonationProjectCard/DonationProjectCard.module.css

```diff
@@ -0,0 +1,33 @@
+.container {
+  height: 240px;
+  position: relative;
+}
+
+.bottombox {
+  bottom: 0;
+  display: flex;
+  flex-direction: row;
+  height: 72px;
+  position: absolute;
+  width: inherit;
+}
+
+.bar {
+  background: var(--color-sheltertech-blue);
+  height: 100%;
+  margin-right: 12px;
+  width: 8px;
+}
+
+.title {
+  color: var(--color-white);
+  font-family: var(--font-family-lato);
+  font-size: var(--font-size-subtitle);
+  font-weight: bold;
+}
+
+.subtitle {
+  color: var(--color-white);
+  font-family: var(--font-family-lato);
```

Comment:
Similar to above, you should use the `font` shorthand variables that we've defined. This one I believe corresponds to `--font-body-small`.

Reply:
👍 Done.